evap/staff/tests/test_importers.py

```diff
@@ -25,8 +26,7 @@ def setUpTestData(cls):
             cls.invalid_excel_content = excel_file.read()
         with open(cls.filename_random, "rb") as excel_file:
             cls.random_excel_content = excel_file.read()
-        with open(cls.filename_duplicate, "rb") as excel_file:
```

Comment:
Just a reminder for your later cleanup that `filename_duplicate` is unused (and I think that no linter will catch this)